cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -976,6 +983,32 @@ public void getServiceInstances(HttpRequest request, HttpResponder responder,
   }
 
   /**
+   * Return the
+   */
+  @GET
+  @Path("/apps/{app-id}/services/{service-id}/active")
+  public void getServiceStatus(HttpRequest request, HttpResponder responder,
+                               @PathParam("namespace-id") String namespaceId,
+                               @PathParam("app-id") String appId,
+                               @PathParam("service-id") String serviceId) throws NotFoundException {
+    ProgramId programId = Ids.namespace(namespaceId).app(appId).service(serviceId);
+    ProgramStatus status = lifecycleService.getProgramStatus(programId);
+    if (status == ProgramStatus.STOPPED) {
+      responder.sendString(HttpResponseStatus.BAD_REQUEST, "Service is stopped. Please start it.");
```

Comment:
Since the client is asking whether a service is active, the right response code should be 503 - stopped, right?